collector/src/bin/rustc-perf-collector/execute.rs

```diff
@@ -88,8 +87,9 @@ pub struct Benchmark {
     config: BenchmarkConfig,
 }
 
-struct CargoProcess<'sysroot> {
-    sysroot: &'sysroot Sysroot,
+struct CargoProcess<'a> {
+    rustc_path: &'a PathBuf,
```

Comment:
hm, these should probably be `&'a Path` instead.

Reply:
I've done that, and likewise for a bunch of the `rustc_path`/`cargo_path` function parameters. It was much easier than I expected... there must be some kind of automatic `PathBuf` to `Path` coercions happening.